JASP-Desktop/widgets/filemenu/actionbuttons.cpp

```diff
@@ -21,12 +21,12 @@ void ActionButtons::loadButtonData(std::vector<ActionButtons::DataRow> &data)
 {
 	_data =
 	{
-		{FileOperation::Open,			tr("Open"),				true,	{ResourceButtons::RecentFiles,	ResourceButtons::Computer,	ResourceButtons::DataLibrary, ResourceButtons::OSF }		},
+		{FileOperation::Open,			tr("Open"),				true,	{ResourceButtons::Computer, ResourceButtons::OSF, ResourceButtons::RecentFiles,	ResourceButtons::DataLibrary }		},
```

Comment:
Why did you change the order?

Reply:
To make it look better and the order doesnt matter anyway I think